examples/minnietwitter/src/main/java/amino/run/appexamples/minnietwitter/MinnieTwitterMain.java

```diff
@@ -163,6 +165,10 @@ private static TwitterManager createTwitterManager(Registry oms, String microSer
                                 DMSpec.newBuilder()
                                         .setName(AtLeastOnceRPCPolicy.class.getName())
                                         .create())
+                        .addDMSpec(
+                                DMSpec.newBuilder()
```

Comment:
TwitterManager has getters(rpc invocations) for getUserManager() and getTagManager() to return stubs for userManager and TagManager microservices respectively. Instead of using ConsensusRSM for TwitterManager, should we use it UserManager or User ?

Reply:
Yes, I'm still getting there.  It doesn't even work properly for TwitterManager yet :-)